crates/turborepo-telemetry/src/events/command.rs

```diff
@@ -151,4 +151,14 @@ impl CommandEventBuilder {
         });
         self
     }
+
+    // Successful/Failed logins
+    pub fn track_login_success(&self, succeeded: bool) -> &Self {
+        self.track(Event {
+            key: "login:success".to_string(),
```

Comment:
The `login` prefix is redundant here because any `command` events (this struct) automatically get a field appended to them with the current command. So that context is already wired up!